specs/_features/eip7732/beacon-chain.md

```diff
@@ -655,6 +655,7 @@ def process_execution_payload(state: BeaconState,
                 execution_payload=payload,
                 versioned_hashes=versioned_hashes,
                 parent_beacon_block_root=state.latest_block_header.parent_root,
+                execution_requests=ExecutionRequests(),  # TODO: fix me (making linter happy)
```

Comment:
@potuz sorry for the little hack... :). We are gonna rebase a bunch of these changes anyway so 🤷